site/en/docs/handbook/components/index.md

```diff
@@ -361,6 +361,10 @@ const config = {
 
 {% endColumns %}
 
+## Comments
+
+If you want comments that are stripped out during rendering, you can place your comment text between {&num; and  &num;}</code>.
```

Comment:
Should this be within code brackets? Otherwise, you should drop `</code>`

Reply:
oops forgot to delete the </code> thanks!  BTW Nothing else works. not code blocks, not <pre>, not <code>. The comments are literally erased. so I used plain text.